.travis.yml

```diff
@@ -36,8 +36,6 @@ install:
 before_script:
   - echo "zend_extension=xdebug.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
 
-script:
-  - vendor/bin/phpunit -c phpunit.xml.dist --coverage-clover build/logs/clover.xml
+script: ./vendor/bin/phpunit tests/YahooWeatherClientTests.php --coverage-html --coverage-clover=coverage.xml
```

Comment:
It;s very strange whem I use ```vendor/bin/phpunit -c phpunit.xml```.
The PHPUnit will not execute the unit testing and output the message: ```No tests executed``` during the travis-ci runtime.
And I got the same result from my local unit testing environment.